portfolio/src/main/webapp/style.css

```diff
@@ -151,3 +153,63 @@ nav ul li a {
 .hidden {
   display: none;
 }
+
+.aboutimg1, .aboutimg2, .aboutimg3, .aboutimg4{
+  position:relative;
+  opacity:1;
+  background-position:center;
+  background-size:cover;
+  background-repeat:no-repeat;
+  background-attachment:fixed;
+}
+
+.aboutimg1{
+  background-image:url('/images/blackbackground.jpg');
+  min-height:100%;
+}
+
+.aboutimg2{
+  background-image:url('/images/longgrove.jpg');
+  min-height:400px;
+}
+
+.aboutimg3{
+  background-image:url('/images/stanford.jpg');
+  min-height:400px;
+}
+
+.aboutimg4{
+  background-image:url('/images/beach.jpg');
+  min-height:400px;
+}
+
+.section{
+  text-align:center;
```

Comment:
nit, space between the attribute and the value. e.g. text-align: center;
also spaces between the class definition and the {